mycli/encodingadaptor.py

```diff
@@ -0,0 +1,29 @@
+class EncodingAdaptor:
+    def __init__(self, cur):
+        self.cur = cur
+
+    def __iter__(self):
+        return iter(self.fetch, None)
+
+    def fetch(self):
+        row = self.cur.fetchone()
+        if row:
+            return [encodeField(x) for x in row]
+        else:
+            raise StopIteration
+
+
+def encodeField(field):
```

Comment:
It is typical to use underscore_separated_names for function names and method names.